### ISSUE TITLE
Because React v16 remove React.PropTypes, react-swf does't work well

### ISSUE THREAD

**Issue by ShangcaoYuan** [syranide/react-swf#17]
When i update react v16, i see this
```
The deprecations introduced in 15.x have been removed from the core package. 
React.createClass is now available as create-react-class, React.PropTypes as prop-types, 
React.DOM as react-dom-factories, react-addons-test-utils as react-dom/test-utils,
and shallow renderer as react-test-renderer/shallow. See 15.5.0 and 15.6.0 blog posts for 
instructions on migrating code and automated codemods.
```
So it not work as below:

```
var PropTypes = React.PropTypes;
ReactSWFCompat.propTypes = {
  container: PropTypes.element.isRequired
};
```
Plz update PropTypes that depends on 'prop-types'

**Comment by syranide** [comment]
Thank you, haven't been hands on with v16 yet but I will look into this.

**Comment by syranide** [comment]
I've published a new version 1.0.7 which should resolve the issue.

**Comment by ShangcaoYuan** [comment]
It's work. Thanks for the work on this!
